sql-ballerina/tests/complex-query-test.bal

```diff
@@ -65,6 +64,60 @@ function testGetPrimitiveTypes() returns error? {
     test:assertEquals(value, expectedData, "Expected data did not match.");
 }
 
+@test:Config {
+    groups: ["query", "query-complex-params"]
+}
+function testGetPrimitiveTypes2() returns error? {
+    MockClient dbClient = check new (url = complexQueryDb, user = user, password = password);
+    stream<SelectTestAlias, error?> streamData = dbClient->query(
+	"SELECT int_type, long_type, double_type,"
+        + "boolean_type, string_type from DataTable WHERE row_id = 1", SelectTestAlias);
```

Comment:
Shall we add a test case, without passing row type?

Reply:
it is already there as `testToJsonComplexTypes()`

Reply:
This test case doesn't test infering the return type. shall we add one

Reply:
like 
```
    stream<SelectTestAlias, error?> streamData = dbClient->query("SELECT int_type, long_type, double_type,"        + "boolean_type, string_type from DataTable WHERE row_id = 1");
```

Reply:
done. Added as `testGetPrimitiveTypes3()`